src/components/compound/DlSlider/components/DlSliderInput.vue

```diff
@@ -62,25 +71,32 @@ export default defineComponent({
             if (sliderInput.value) sliderInput.value.value = value
         }
 
-        const debouncedHandleChange = computed(() => {
-            if (stateManager.disableDebounce) {
```

Comment:
@jabezX why did you remove the debounce ?

I saw the comment about since changing mode to text you needed it to mitigate having non numeric characters input 

If you wanted that you couldve just used the `inputmode="numeric"` on the input. that solves it. 
The basic html input 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number
should accept numeric as . and - by default. its weird that we have to switch type to accept those in our case

in anycase having inputMode set to numeric should fix it for you too. we need the debounce as to not spam the events